Stresser/RegistryArtifact.cpp

```diff
@@ -0,0 +1,70 @@
+
+#include "RegistryArtifact.h"
+
+RegistryArtifact::RegistryArtifact(std::string id, std::string name, std::string type, std::string data) : RuleEntity(id, name, type, data) 
+{
+	this->install();
+}
+
+RegistryArtifact::~RegistryArtifact()
+{
+	this->unInstall();
+}
+
+void RegistryArtifact::install() const
+{
+	wil::unique_hkey hKey;
```

Comment:
We don't use WIL anymore
I will add AutoRegKey class / send you ready one to add by yourself (to Utils static library)